src/config/db.config.js

```diff
@@ -0,0 +1,23 @@
+const dotenv = require('dotenv');
```

Comment:
Please use ES6 import here.

Reply:
ES6 import throws an error. I have tried to fix it using babel as it was specified in the documentation, but it still throws the error. 
I am working on it now though. If nothing seems to work after an hour or two, I will let you know.